lib/sles4sap/sdaf_deployment_library.pm

```diff
@@ -827,6 +828,48 @@ sub get_sdaf_deployment_command {
     return $cmd;
 }
 
+=head2 git_clone
+
+    git_clone('https://github.com/myrepo/tree/main' [, branch=>'development', quiet=>'1', skip_ssl_verification=>'true',
+        output_log_file=>'git_clone.log']);
+
+B<repository>: Git repository url. Mandatory argument.
+
+B<branch>: Clone specific branch. Default: not defined
+
+B<quiet>: Minimize output verbosity. Default: false
+
+B<skip_ssl_verification>: Enable or disable SSL related error. Useful in case of self signed certificates. Default: false
```

Comment:
Yes. I was writing this to be a general function, so I check what other usages are there. So this could be used while cloning repo with self signed SSL certs. Disabling ssl check globally is discouraged.
In any case, the option is disabled by default so unless you specify it by caller it's won't be included in cmd. The function will not affect current usage.